client/e2e/utils/enums.ts

```diff
@@ -60,18 +222,42 @@ export enum OperatorUUID {
   DEFAULT = "4242ea9d-b917-4129-93c2-db00b7451051",
 }
 
+// 📝 Status LOV Operation
+export enum OperationStatus {
+  PENDING = "Pending",
+  APPROVED = "Approved",
+  DECLINED = "Declined",
```

Comment:
nit: I don't think we need to write these out again, we have them in `@/app/utils/enums.ts`